src/app/search/directives/story-list.component.css

```diff
@@ -0,0 +1,49 @@
+publish-story-card, .story {
+  display: block;
+  overflow: hidden;
+  position: relative;
+  margin-top: 20px;
+  margin-right: 20px;
+  flex-basis: 320px;
+  -ms-flex-preferred-size: 320px;
+  -webkit-flex-basis: 320px;
+  -webkit-box-flex: 3;
+  -moz-box-flex: 3;
+  -o-box-flex: 3;
+  -ms-box-flex: 3;
+  box-flex: 3;
+  -webkit-flex-grow: 3;
+  flex-grow: 3;
+  -webkit-flex-shrink: 1;
+  flex-shrink: 1;
+  height: 200px;
+  background: #fff;
+}
+
+/* not a great attempt to fix the odd last items stretching
```

Comment:
similar to last row odd items stretching for series only worse for stories because the base size has 3 items in a row